README.md

```diff
@@ -114,8 +114,6 @@ Below, please find a list of all members and their organizations.
 
 ## **Join CASA**
 
-CASA will begin meetings in 2021 as we wait for the initial working groups to form.
-
-1. Submit a Pull Request to this repository adding yourself as a member in the table above.
+Submit a Pull Request to this repository adding yourself as a member in the table above.
 
 Once a maintainer has confirmed you have completed this step, they will merge your PR and send you a welcome email containing your invitations to the **CASA Call Calendar** and **Discord**.
```

Comment:
```suggestion
Once a maintainer has confirmed you have completed this step, they will merge your PR.  We recommend checking that your GH email is up-to-date and "Watching" all issues and pull requests in the /CASA/  repo, as well as /namespaces/ and/or /CAIPs/ to stay up-to-date.
```

Reply:
The Gatherings are in the Gatherings repo - maybe we should ask to watch this one (also)?